content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -6,6 +6,35 @@ weight: 5
 
 This section provides specific deployment or configuration guidelines to avoid networking or traffic management issues.
 
+## 503 errors after setting destination rule
+
+If, after applying a `DestinationRule` for a service, requests to the service immediately start generating HTTP 503 errors
+and continue to do so until the `DestinationRule` is removed or reverted, it is possible that the `DesintationRule`
+has caused a TLS conflict for the service.
```

Comment:
Keep conditional clauses together to improve the flow of text and to simplify punctuation. To simplify for clarity:
If requests to the service immediately start generating HTTP 503 errors after you applied a `DestinationRule` for a service and the errors continue until you remove or revert the `DestinationRule`, then the `DesintationRule` probably causes a TLS conflict for the service.